src/pages/index.js

```diff
@@ -265,12 +265,12 @@ function handleAvatarSubmit(evt) {
   api
     .editAvatarInfo(avatarInput.value)
     .then(() => {
-      (profileAvatar.src = avatarInput.value),
-        disableButton(avatarSubmitButton, settings),
-        closeModal(addAvatarModal);
+      profileAvatar.src = avatarInput.value;
+      disableButton(avatarSubmitButton, settings);
+      closeModal(addAvatarModal);
     })
-    .catch(console.error)
-    .finally((submitButton.textContent = "Save"));
+    .catch(() => console.error)
```

Comment:
` .catch(console.error);` should be used instead of `.catch(() => console.error)`.  


It should be either `.catch(() => console.error());` or ` .catch(console.error);`, but not  `.catch(() => console.error)` because the `console.error` is not called inside the arrow function, so it doen't work


**Please, correct it everywhere in the code**